src/components/AdminSettings.vue

```diff
@@ -63,23 +62,11 @@ function submitForm (data) {
 
       <v-col class="pt-15">
       <h3 class="font-weight-bold text-red-accent-4" >Delete Account</h3>
-      <v-divider class="redColor" />
+      <v-divider color="red-accent-4"  />
```

Comment:
I think we should use the "error" color (or a similarly transformed value) in this and similar cases https://vuetifyjs.com/en/features/theme/#customizing